addons/im_livechat/static/src/core/public_web/messaging_menu_patch.js

```diff
@@ -14,6 +14,11 @@ patch(MessagingMenu.prototype, {
         );
         if (hasLivechats) {
             items.push({
+                counter: this.store.discuss.livechats.reduce(
+                    (acc, channel) =>
+                        channel.selfMember?.message_unread_counter > 0 ? acc + 1 : acc,
+                    0
+                ),
```

Comment:
There is no `getDiscussSidebarCategoryCounter` for livechat?

Reply:
No, livechats are split into their own categories in sidebar. There's a default one but as a default it has the livechats that are not in named category.

Reply:
Maybe take the counter of all categories combined? It feels better than repeating the condition about what is considered an unread value for them.

Reply:
We don't have a way to determine whether a category is from livechat: the name and id is custom, and it can be empty with no thread.

We could say it's the exclusion of all other categories, but I don't think it's a good idea to have some other patch that other categories need to help livechat for.